contracts/UniversalTokenRouter.sol

```diff
@@ -137,7 +138,12 @@ contract UniversalTokenRouter is IUniversalTokenRouter {
             if (token.id == EIP_721_ALL) {
                 return IERC721(token.adr).balanceOf(owner);
             }
-            return IERC721(token.adr).ownerOf(token.id) == owner ? 1 : 0;
+            try IERC721(token.adr).ownerOf(token.id) {
```

Comment:
why do we need this? if the id is invalid, it still revert, right?

Reply:
if ID doesn't exist, IERC721(token.adr).ownerOf(token.id) will throw an error and revert tx
so I put try cacth to return 0 for _balanceOf function in this case

Reply:
no need for `ERC721_INVALID_TOKEN_ID` check